src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/ListViewRenderer.cs

```diff
@@ -366,6 +366,8 @@ void UpdateFooterMeasure()
 			var platformFrame = new RectangleF(0, 0, size.Width, size.Height);
 			_footerRenderer.PlatformView.Frame = platformFrame;
 			_footerRenderer.VirtualView.Arrange(platformFrame.ToRectangle());
+			Control.TableFooterView = _footerRenderer.PlatformView;
+
 			BeginInvokeOnMainThread(() =>
 			{
 				if (_headerRenderer != null)
```

Comment:
```suggestion
				if (_footerRenderer != null)
```

If I'm reading this code correctly. I think the fix is right here. This looks like a copy paste mistake from the `UpdateHeaderMeasure` code.

Reply:
Good catch.
But the UpdateHeaderMeasure update Control.TableHeaderView twice (one in main thread), not sure it is necessary in footer. 
But I tested with just if (_footerRenderer != null), that is enough to fix the problem. Control.TableFooterView = _footerRenderer.PlatformView; might not be needed. Maybe the Control.TableHeaderView = _headerRenderer.PlatformView; in UpdateHeaderMeasure might not be needed as well.